fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirSchemaVersion.java

```diff
@@ -26,6 +26,7 @@
     ,V0009(9, "issue-1683 refactor composite values")
     ,V0010(10, "issue-1958 add IS_DELETED flag to each xxx_LOGICAL_RESOURCES table")
     ,V0011(11, "issue-2011 add LAST_UPDATED to each xxx_LOGICAL_RESOURCES table")
+    ,V0012(12, "issue-2032 change column order for index on RESOURCE_CHANGE_LOG")
```

Comment:
Do we need to set the version of the RESOURCE_CHANGE_LOG table to v12 now?  Is that change included in this PR?

Reply:
Maybe this change snuck in here but should have been saved for a future PR that addresses this issue?

Reply:
Good catch. I backed out the change after realizing the issue wasn't with the index column order, but with the sort columns in the query. I'll remove V0012 because it's not used.